src/app/components/Image/index.tsx

```diff
@@ -0,0 +1,125 @@
+/** @jsx jsx */
+import React from 'react';
+import { jsx } from '@emotion/react';
+import { Helmet } from 'react-helmet';
+import styles from './index.styles';
+
+interface Props {
+  alt: string;
+  aspectRatio?: number;
+  className?: string;
+  fallbackMediaType: string;
+  fallbackSrcset: string;
+  height: number;
+  isAmp?: boolean;
+  lazyLoad?: boolean;
+  placeholder?: boolean;
+  preload?: boolean;
+  primaryMediaType: string;
+  primarySrcset: string;
+  sizes: string;
+  src: string;
+  width: number;
+}
+
+const DEFAULT_ASPECT_RATIO = 16 / 9;
+
+const Image = ({
+  alt,
+  aspectRatio,
+  className,
+  fallbackMediaType,
+  fallbackSrcset,
+  height,
+  isAmp = false,
+  lazyLoad = false,
+  placeholder = true,
+  preload = false,
+  primaryMediaType,
+  primarySrcset,
+  sizes = '100vw',
+  src,
+  width,
+}: React.PropsWithChildren<Props>) => {
+  const wrapperAspectRatio =
+    aspectRatio || width / height || DEFAULT_ASPECT_RATIO;
+
+  return (
+    <React.Fragment>
+      {preload && (
+        <Helmet>
+          <link
+            rel="preload"
+            as="image"
+            href={src}
+            imagesrcset={primarySrcset}
+            imagesizes={sizes}
+          />
+        </Helmet>
+      )}
+      <div
+        className={className}
+        css={[
+          styles.wrapper,
+          {
+            paddingBottom: `${(1 / wrapperAspectRatio) * 100}%`,
+          },
+        ]}
+      >
+        <div
+          css={[placeholder && styles.placeholder, styles.placeholderWrapper]}
+        >
+          {isAmp ? (
+            <amp-img
+              alt={alt}
```

Comment:
good old amp, are there proper types for amp attributes?

Reply:
There aren't typescript types for AMP but we're looking at a few workarounds so PR isn't ready for full review quite just yet

Reply:
Following the advice [here](https://nextjs.org/docs/advanced-features/amp-support/typescript) I've addressed this [here](https://github.com/bbc/simorgh/pull/10265/files#diff-e044a0734938281ee2184aeb22e60ae3502717d53b8f522328ac7ec4685a459b)